train.py

```diff
@@ -92,23 +92,33 @@ def main(job_config: Any):
         special_mode=job_config.dataloader.special_mode,
     )
 
+    if not job_config.validation.batch_size:
+        logger.info("Validation batch size not specified, training batch size is used.")
+        job_config.validation.batch_size = job_config.training.batch_size
+
+    # build validation dataloader
+    valid_data_loader = build_hf_data_loader(
+        job_config.validation.dataset,
+        job_config.validation.dataset_path,
+        job_config.training.data_processing_style,
+        tokenizer,
+        job_config.validation.batch_size,
+        job_config.training.seq_len,
+        dp_degree,
+        dp_rank,
+        False,
+        pin_memory=job_config.dataloader.pin_memory,
+        num_workers=job_config.dataloader.num_workers,
+        special_mode=job_config.dataloader.special_mode,
+    )
+
     # check the validation parameters
-    if not job_config.validation.dataset_path and not job_config.validation.dataset:
+    if not job_config.validation.dataset:
```

Comment:
let's move this to the config_manager and only raise the error if validation is enabled.